docs/api-reference/geo-layers/tile-layer.md

```diff
@@ -179,13 +192,17 @@ How the tile layer refines the visibility of tiles. One of the following:
 
 ##### `maxRequests` (Number, optional)
 
-The maximum number of concurrent `getTileData` calls. 
+The maximum number of concurrent `getTileData` calls.
+
+If `<= 0`, no throttling will occur, and `getTileData` may be called an unlimited number of times concurrently regardless of how long that tile is or was visible.
+
+If `> 0`, a maximum of `maxRequests` instances of `getTileData` will be called concurrently. Requests may never be called if the tile wasn't visible long enough to be scheduled and started. Requests may also be aborted (through the `signal` passed to `getTileData`) if there are more than `maxRequests` ongoing requests and some of those are for tiles that are no longer visible.
 
-If `<= 0`, no throttling will occur, and `getTileData` may be called an unlimited number of times concurrently.
+If `getTileData` makes `fetch` requests against an HTTP 1 web server, then `maxRequests` should correlate to the browser's maximum number of concurrent `fetch` requests. For Chrome, the max is 6 per domain. If you use the `data` prop and specify multiple domains, you can increase this limit. For example, with Chrome and 3 domains specified, you can set `maxRequests=18`.
 
-If `> 0`, a maximum of `maxRequests` instances of `getTileData` will be called concurrently. Additionally, for requests that are throttled, if the tile is no longer visible in the viewport when a request slot opens up, that request will be cancelled. When requests are not throttled, `getTileData` will be called for all visible tiles, regardless of how long that tile is or was visible.
+If the web server supports HTTP/2 (Open Chrome dev tools and look for "h2" in the Protocol column), then you can make an unlimited number of concurrent requests (and can set `maxRequests=-1`).
```

Comment:
I might add "Note that this will request data for every tile, no matter for how long the tile was visible, and may increase server load."